packages/strapi-admin/admin/src/components/LeftMenu/LeftMenuLink/A.js

```diff
@@ -10,7 +11,6 @@ const A = styled.a`
   cursor: pointer;
   color: ${props => props.theme.main.colors.leftMenu['link-color']};
   text-decoration: none;
-  display: block;
   -webkit-font-smoothing: antialiased;
 
   &:hover {
```

Comment:
I put this comment here but it is not related to this file. 

Something is wrong with the menu link hover background (Screenshot).

The way to fix that : 
1) Go to `/strapi-admin/admin/src/components/LeftMenu/LeftMenuLinkSection/LeftMenuListLink.js`
2) Remove the `margin-right: 28px;`
3) Go to `/strapi-admin/admin/src/components/LeftMenu/LeftMenuLink/A.js`
4) Add `padding-right: 2.3rem;` to the `&.linkActive` line 34.

![Capture d’écran 2020-10-15 à 14 48 30](https://user-images.githubusercontent.com/7756284/96131153-1d878b80-0ef9-11eb-9cc5-f502177a2024.png)